assets/src/containers/ResourcesAccessed.js

```diff
@@ -228,21 +229,25 @@ function ResourcesAccessed (props) {
   }
 
   const resourceAccessChartBuilder = (resourceData, resourcesLimit) => {
+    const resourceDataSorted = resourceData.sort((a, b) => b.total_percent - a.total_percent)
     if (resourceTypeFilter.length === 0) {
       return (<AlertBanner>Please select a resource type to display data.</AlertBanner>)
-    } else if (!resourceData || Object.keys(resourceData).length === 0) {
+    } else if (!resourceDataSorted || Object.keys(resourceDataSorted).length === 0) {
       return (<AlertBanner>Resource data for your selections is not available.</AlertBanner>)
     } else {
       return (
         <>
```

Comment:
This seems to be causing a problem when there is no data returned (I found this because I haven't run the cron recently). Basically, `resourceData` seems to be an object when no data is returned, and an array when it is. This is confusing to me, but perhaps out of scope. I would probably just move the sorting into the `else` clause, like so:

```
    if (resourceTypeFilter.length === 0) {
      return (<AlertBanner>Please select a resource type to display data.</AlertBanner>)
    } else if (!resourceData || Object.keys(resourceData).length === 0) {
      return (<AlertBanner>Resource data for your selections is not available.</AlertBanner>)
    } else {
      const resourceDataSorted = resourceData.sort((a, b) => b.total_percent - a.total_percent)
      return (
        <>
```